server/src-lib/Hasura/RQL/DDL/Permission/Triggers.hs

```diff
@@ -44,4 +44,5 @@ buildInsTrigFn fn tn be = do
     tmpltVals = J.object [ "function_name" J..= toSQLTxt fn
                          , "table_name" J..= toSQLTxt tn
                          , "check_expression" J..= toSQLTxt be
+                         , "check_expression_literal" J..= pgFmtLit (toSQLTxt be)
```

Comment:
Do not use `pgFmtLit` or `pgFmtIden`, except in modules where it is defined. 
Here you can make use of `SELit` constructor.

Reply:
Done